controllers/categories.controller.js

```diff
@@ -0,0 +1,13 @@
+const {
+  fetchCategories,
+} = require(`${__dirname}/../models/categories.model.js`);
+
+exports.getCategories = (req, res, next) => {
+  fetchCategories()
+    .then((category) => {
+      res.status(200).send({ categories: category });
```

Comment:
Technically the value we are sending back here is an array of categories so the name 'category' isn't quite accurate. A minor thing but lets be as descriptive as possible :)